pom.xml

```diff
@@ -22,9 +22,7 @@
 	</properties>
 
 	<modules>
-		<module>samples-giis-template-m1</module>
-		<module>samples-giis-template-m2</module>
-		<module>samples-giis-template-bom</module>
+		<module>eshopcontainers-e2etestsuite</module>
```

Comment:
Do we really need a multimodule project? if not, convert into single module project